CHARTER.md

```diff
@@ -0,0 +1,73 @@
+# Express Charter
+
+## Section 0: Guiding Principles
+
+The Express project is part of the OpenJS Foundation which operates transparently, openly, collaboratively, and ethically.
+Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+## Section 1: Scope
+
+Express is a web server framework with a simple and expressive API which is highly aligned with Node.js core.
+We aim to be be the best in class for writing performant, spec compliant and powerful web servers in Node.js.
+As one of the oldest and most popular web frameworks in the ecosystem, we have an important place for new
+users and experts alike.
+
+### 1.1: In-scope
+
+Express is made of many modules spread between three GitHub Orgs:
+
+- [expressjs](http://github.com/expressjs/): Top level middleware and libraries
+- [pillarjs](http://github.com/pillarjs/): Components which make up Express but can also be used for other web frameworks
+- [jshttp](http://github.com/jshttp/): Low level http libraries
+
+### 1.2: Out-of-Scope
+
+Section Intentionally Left Blank
```

Comment:
I'm not sure if we should try to define some out-of-scope things. Perhaps we increase the pluggability and extendibility of the framework above and then we could mention some things as out of scope in which users could write as extensions? I'm trying to better articulate my thoughts, but I'm sure that we do have out-of-scope items we could add. Maybe even like "WebSockets" could be added here to get us started?

Reply:
I also realized after typing that I may be misunderstanding what in-scope / out-of-scope mean in the context of this document.

Reply:
I think the scope here is the projects, modules and repos.  I could be wrong, but I was going off of the example their template linked to:

https://github.com/kubernetes/community/blob/master/sig-architecture/charter.md#out-of-scope